python/python/glide/constants.py

```diff
@@ -33,8 +33,29 @@
 TSingleNodeRoute = Union[RandomNode, SlotKeyRoute, SlotIdRoute, ByAddressRoute]
 # When specifying legacy path (path doesn't start with `$`), response will be T
 # Otherwise, (when specifying JSONPath), response will be List[Optional[T]].
+#
+# Explanation:
+# TJsonResponse is designed to handle scenarios where some paths may not contain valid values, especially with JSONPath targeting multiple paths.
+# In such cases, the response may include None values, represented as `Optional[T]` in the list.
+# This type provides flexibility for commands where a subset of the paths may return None.
+#
 # For more information, see: https://redis.io/docs/data-types/json/path/ .
```

Comment:
Is there non Redis docs to refer to? Maybe ValKey or AWS offering something? Some blog-post of someone random?

Reply:
didnt find in valkey, I will search in aws

Reply:
Is it a good cover? If so, might be better, I think. In general.